components/list/list-item-mixin.js

```diff
@@ -103,14 +88,18 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			_highlighting: { type: Boolean, reflect: true },
 			_hasNestedList: { state: true },
 			_siblingHasColor: { state: true },
-			_slimColor: { state: true }
 		};
 	}
 
 	static get styles() {
 
 		const styles = [ css`
 			:host {
+				--d2l-list-item-color-border-radius: var(--d2l-list-color-border-radius, 6px);
+				--d2l-list-item-color-width: var(--d2l-list-color-width, 6px);
+				--d2l-list-item-illustration-margin-inline-end: var(--d2l-list-illustration-margin-inline-end, 0.9rem);
+				--d2l-list-item-illustration-max-height: var(--d2l-list-illustration-max-height, 2.6rem);
+				--d2l-list-item-illustration-max-width: var(--d2l-list-illustration-max-width, 4.5rem);
```

Comment:
Let me know if I'm doing this totally wrong. It seems to work but I could see this not being done in a best practice way.

Reply:
Do we need these `--d2l-list-item-*` variables? i.e. can the styles below just use the variables inherited from and defined by `d2l-list`?

Reply:
Ah yup seems to work without - will fix that!